core-plugins/docs/RowDenormalizer-batchaggregator.md

```diff
@@ -10,39 +10,60 @@ Use Case
 The transform takes input record that stores a variable set of custom attributes for an entity, denormalizes it on the basis of the key field, and then returns a denormalized table according to the output schema specified by the user.
 The denormalized data is easier to query.
 
+Conditions
+----------
+In case a field value is not present, then it will be considered as NULL.
+
+For Example,
+
+If keyfield('id') in the input record is NULL, then that particular row will not be considered.
+
+If fieldname('attribute') and fieldvalue('value') is not present, then denormalized output will have NULL for that
```

Comment:
if the field name is not present, how can there be an output field? Do you mean if the fieldname is null the field will not exist in the output?